sync/storage/batch/batch.go

```diff
@@ -36,7 +45,7 @@ func (s *batchStorageSync) Sync(ctx context.Context, lastSuccessfulRun time.Time
 
 	ch := make(chan *syncError)
 	for _, b := range buckets {
-		go s.syncBucket(ctx, lastSuccessfulRun, b.Name, ch)
+		go s.syncBucket(ctx, lastSuccessfulRun, b.Name, ch, bucketRateLimit)
```

Comment:
invalid operation: b (variable of type *invalid type) has no field or method Name